### PR TITLE
New list filtering defaults and item newness commands

### PR DESCRIPTION
**Pull request by jameshaydon**
Updates to docs for https://github.com/radicle-dev/radicle/pull/572 and https://github.com/radicle-dev/radicle/pull/577.

### CODE REVIEW

File: content/docs/rad-cli-reference.rst [hunk 1]
```diff
@@ -63,6 +64,7 @@ rad patch commands
                       <state2> ...'.
                       If the option '--fancy' is appended, the list is opened via
                       fzf (note that fzf has to be installed).
+                      Unread patches are marked with a * before the timestamp.
        propose      - Create a new patch from a single commit
```
Comment (juliendonck) [inline content/docs/rad-cli-reference.rst]: should be `after the timestamp`

File: content/docs/tutorial-part-1-contributor.rst [hunk 2]
```diff
@@ -109,7 +109,7 @@ Like our git repo, Radicle stores our issues and patches locally and distributes
 
   $ rad issue list
   state    #   title                         author    updated
-  open     0   Patch proposal not accepted   jane      2019-03-04T16:41:47Z
+  open     0   Patch proposal not accepted   jane      2019-03-04T16:41:47Z *
```
Comment (juliendonck) [inline content/docs/tutorial-part-1-contributor.rst]: Am I understanding this correctly that it's just added to the `updated` column and not it's own `unread` column?

Comment (jameshaydon) [reply content/docs/tutorial-part-1-contributor.rst]: Indeed, feel free to propose something else though.

File: content/docs/rad-cli-reference.rst (hunk 1 above)
Comment (jameshaydon) [reply content/docs/rad-cli-reference.rst]: https://github.com/radicle-dev/radicle/pull/580

File: content/docs/rad-cli-reference.rst [hunk 3]
```diff
@@ -33,25 +34,32 @@ rad issue commands
...
+       all-read     - Mark all issues as read
        help         - Print this help and exit
 
 rad patch commands
 ==================
 
 .. code-block::
-
-  rad patch - Radicle patch CLI
+
```
Comment (xla) [inline content/docs/rad-cli-reference.rst]: Looks like unnecessary whitespace.

File: content/docs/tutorial-part-1-contributor.rst [hunk 4]
```diff
@@ -106,10 +106,12 @@ What lovely gardens. It looks like each garden file contains an ASCII image. The
 Like our git repo, Radicle stores our issues and patches locally and distributes them over the IPFS network. To look at a list of issues we'll run ``$ rad issue list`` while inside the git repo.
 
 ::
-
+
```
Comment (xla) [inline content/docs/tutorial-part-1-contributor.rst]: Unnecessary indentation.

File: content/docs/tutorial-part-2-maintainer.rst [hunk 5]
```diff
@@ -66,10 +66,12 @@ Let's say a collaborator already proposed a patch using the steps outlined in th
 We can see a list of all proposed patches, as well as the status of each patch, with ``$ rad patch list``.
 
 ::
-
+
```
Comment (xla) [inline content/docs/tutorial-part-2-maintainer.rst]: Uneccessary whitespace.
